diff -urN box2d-3.1.1.orig/src/CMakeLists.txt box2d-3.1.1/src/CMakeLists.txt
--- box2d-3.1.1.orig/src/CMakeLists.txt	2025-06-04 02:50:24.000000000 +0200
+++ box2d-3.1.1/src/CMakeLists.txt	2025-09-20 03:22:10.964082404 +0200
@@ -171,7 +171,7 @@
 	endif()
 elseif (UNIX)
 	message(STATUS "Box2D using Unix")
-	target_compile_options(box2d PRIVATE -Wmissing-prototypes -Wall -Wextra -pedantic -Wno-unused-value)
+	target_compile_options(box2d PRIVATE -Wmissing-prototypes -Wall -Wextra -pedantic -Wno-unused-value -Wno-maybe-uninitialized)
 	if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "aarch64")
 		# raspberry pi
 		# -mfpu=neon
