diff -urN box2d-3.1.0.orig/src/CMakeLists.txt box2d-3.1.0/src/CMakeLists.txt
--- box2d-3.1.0.orig/src/CMakeLists.txt	2025-04-20 06:12:49.000000000 +0200
+++ box2d-3.1.0/src/CMakeLists.txt	2025-04-27 19:52:29.548057489 +0200
@@ -168,7 +168,7 @@
 	endif()
 elseif (UNIX)
 	message(STATUS "Box2D using Unix")
-	target_compile_options(box2d PRIVATE -Wmissing-prototypes -Wall -Wextra -pedantic -Werror -Wno-unused-value)
+	target_compile_options(box2d PRIVATE -Wmissing-prototypes -Wall -Wextra -pedantic -Werror -Wno-unused-value -Wno-maybe-uninitialized)
 	if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "aarch64")
 		# raspberry pi
 		# -mfpu=neon
