diff -urN glfw-3.5.1.orig/src/nsgl_context.m glfw-3.5.1/src/nsgl_context.m
--- glfw-3.5.1.orig/src/nsgl_context.m	2026-07-31 14:46:04.000000000 +0200
+++ glfw-3.5.1/src/nsgl_context.m	2026-08-02 14:56:30.627434497 +0200
@@ -199,7 +199,6 @@
     NSOpenGLPixelFormatAttribute attribs[40];
     int index = 0;
 
-    ADD_ATTRIB(NSOpenGLPFAAccelerated);
     ADD_ATTRIB(NSOpenGLPFAClosestPolicy);
 
     if (ctxconfig->nsgl.offline)
