diff -urN glfw-3.3.1-orig/src/nsgl_context.m glfw-3.3.1/src/nsgl_context.m
--- glfw-3.3.1-orig/src/nsgl_context.m	2020-01-01 17:24:44.000000000 +0100
+++ glfw-3.3.1/src/nsgl_context.m	2020-01-16 23:39:39.111035190 +0100
@@ -205,7 +205,6 @@
     NSOpenGLPixelFormatAttribute attribs[40];
     int index = 0;
 
-    addAttrib(NSOpenGLPFAAccelerated);
     addAttrib(NSOpenGLPFAClosestPolicy);
 
     if (ctxconfig->nsgl.offline)
