diff -urN glfw-3.3.3.orig/src/nsgl_context.m glfw-3.3.3/src/nsgl_context.m
--- glfw-3.3.3.orig/src/nsgl_context.m	2021-02-17 21:23:27.000000000 +0100
+++ glfw-3.3.3/src/nsgl_context.m	2021-02-28 19:18:10.000000000 +0100
@@ -205,7 +205,6 @@
     NSOpenGLPixelFormatAttribute attribs[40];
     int index = 0;
 
-    addAttrib(NSOpenGLPFAAccelerated);
     addAttrib(NSOpenGLPFAClosestPolicy);
 
     if (ctxconfig->nsgl.offline)
