diff --git a/src/core/windows/SDL_xinput.h b/src/core/windows/SDL_xinput.h
index 85e97ff..c5b2b70 100644
--- a/src/core/windows/SDL_xinput.h
+++ b/src/core/windows/SDL_xinput.h
@@ -211,17 +211,6 @@ typedef struct

 #endif /* HAVE_XINPUT_H */

-/* This struct is not defined in XInput headers. */
-typedef struct _XINPUT_CAPABILITIES_EX
-{
-    XINPUT_CAPABILITIES Capabilities;
-    WORD VendorId;
-    WORD ProductId;
-    WORD ProductVersion;
-    WORD unk1;
-    DWORD unk2;
-} XINPUT_CAPABILITIES_EX, *PXINPUT_CAPABILITIES_EX;
-
 /* Forward decl's for XInput API's we load dynamically and use if available */
 typedef DWORD(WINAPI *XInputGetState_t)(
     DWORD dwUserIndex,      /* [in] Index of the gamer associated with the device */
diff --git a/src/joystick/windows/SDL_xinputjoystick.c b/src/joystick/windows/SDL_xinputjoystick.c
index 44d3457..d5bafd9 100644
--- a/src/joystick/windows/SDL_xinputjoystick.c
+++ b/src/joystick/windows/SDL_xinputjoystick.c
@@ -113,7 +113,7 @@ static SDL_bool GetXInputDeviceInfo(Uint8 userid, Uint16 *pVID, Uint16 *pPID, Ui
         *pPID = capabilities.ProductId;
     }
     if (pVersion) {
-        *pVersion = capabilities.ProductVersion;
+        *pVersion = capabilities.VersionNumber;
     }
     return SDL_TRUE;
 }
